### PR TITLE
Internal testnet deploy

### PR DESCRIPTION
**Pull request by chrdevmar**
making this so it's easy to compare changes to deployment script

### CODE REVIEW

File: deploy/ArbitrumRinkebyDeploy.js [hunk 1]
```diff
@@ -169,101 +143,166 @@ module.exports = async (hre) => {
...
+        changeInterval: "0",
     }
 
     const deploymentData = [
         deploymentData1,
         deploymentData2,
         deploymentData3,
         deploymentData4,
-        deploymentData5,
+        // deploymentData5,
```
Comment (sporejack) [inline deploy/ArbitrumRinkebyDeploy.js]:
```suggestion
```

File: deploy/ArbitrumRinkebyDeploy.js [hunk 2]
```diff
@@ -169,101 +143,166 @@ module.exports = async (hre) => {
...
     ]
 
+    console.log(`Deployed PoolFactory: ${factory.address}`)
+    console.log(`Deployed PoolSwapLibrary: ${library.address}`)
+    console.log(`Deploy PoolKeeper: ${poolKeeper.address}`)
+    console.log(`Deployed TestToken: ${token.address}`)
+    // console.log(`Deployed OracleWrapper: ${oracleWrapper.address}`)
     // console.log(`Deploy PoolKeeper: ${poolKeeper.address}`)
```
Comment (sporejack) [inline deploy/ArbitrumRinkebyDeploy.js]:
```suggestion
```

File: deploy/ArbitrumRinkebyDeploy.js [hunk 3]
```diff
@@ -169,101 +143,166 @@ module.exports = async (hre) => {
...
     for (var i = 0; i < deploymentData.length; i++) {
         let receipt = await execute(
             "PoolFactory",
             {
                 from: deployer,
                 log: true,
+                gasLimit: 10000000,
```
Comment (sporejack) [inline deploy/ArbitrumRinkebyDeploy.js]: Constant?

**Review by sporejack — CHANGES_REQUESTED** [review]

File: deploy/ArbitrumRinkebyDeploy.js [hunk 4]
```diff
@@ -169,101 +143,166 @@ module.exports = async (hre) => {
...
+        "EthUsdSMAOracle",
+        {
+            from: deployer,
+            log: true,
+        },
+        "poll"
+    )
+
+    // deploy BTC SMA Oracle
+    // Note: if SMA oracle is not 24 periods you can't deploy oracle and pool in one go. Will need to wait until SMA oracle has enough periods before deploying pool.
```
Comment (CalabashSquash) [inline deploy/ArbitrumRinkebyDeploy.js]: Is this still true with the ramping period?

**Review by CalabashSquash — CHANGES_REQUESTED** [review]
Do we wanna deploy L2Encoder here? I guess it's only really needed for local testing but it could be good to deploy anyway for testnet testing.

File: deploy/ArbitrumRinkebyDeploy.js (hunk 4 above)
Comment (chrdevmar) [reply deploy/ArbitrumRinkebyDeploy.js]: yeah pretty sure this is no longer true, will also add L2Encoder

**Review by CalabashSquash — APPROVED** [review]

**Review by sporejack — APPROVED** [review]

**Pull request merged**